components/src/core/Drawer/DrawerNavGroup/DrawerNavGroup.tsx

```diff
@@ -120,7 +120,7 @@ const DrawerNavGroupRender: React.ForwardRefRenderFunction<unknown, DrawerNavGro
         className: userClassName,
         items = [],
         title,
-        titleColor = theme.palette.text.primary,
+        titleColor = theme.vars.palette.text.primary,
```

Comment:
```suggestion
        titleColor = (theme.vars || theme).palette.text.primary,
```

Reply:
You get the idea...you can do this everywhere to (hopefully) maintain backwards compatibility with v5 theme